source/layouts/section_reference.haml

```diff
@@ -6,7 +6,7 @@
 
   - if current_page.data.table_of_contents
     - content_for :complementary do
-      %h3 Page Sections
 
       %nav.page-sections.sl-c-list-navigation-wrapper.sl-c-list-navigation-wrapper--collapsible
+        %h3 Page Sections
```

Comment:
Style preference: do you mind adding a carriage return between the h3 and the current_page line?

Reply:
So you mean something like this?
```
 %nav.page-sections.sl-c-list-navigation-wrapper.sl-c-list-navigation-wrapper--collapsible
        %h3 Page Sections
        
        = current_page.metadata[:table_of_contents]
```

Reply:
yes, thanks!